public/docs/dart/latest/tutorial/toh-pt4.jade

```diff
@@ -59,20 +60,20 @@ code-example(language="bash").
   First, defining heroes is not the component's job.
   Second, we can't easily share that list of heroes with other components and views.
 
-  We can refactor this hero data acquisition business to a single service that provides heroes and
+  We can refactor this hero data acquisition business to a single service that provides heroes, and
   share that service with all components that need heroes.
 
   ### Create the HeroService
   Create a file in the `lib` folder called `hero_service.dart`.
 .l-sub-section
   :marked
     We've adopted a convention in which we spell the name of a service in lowercase followed by `_service`.
-    If the service name were multi-word, we'd spell the base filename with lower underscore case (AKA "snake_case").
+    If the service name were multi-word, we'd spell the base filename in lower underscore case (also called [snake_case](../guide/glossary.html#!#snake_case)).
```

Comment:
Fix the snake_case link? (It just went to the top of the glossary page, for me.)